core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1560,10 +1596,20 @@ class KafkaController(val config: KafkaConfig,
         // Used only in test cases
         case event: MockEvent =>
           event.process()
+        case ShutdownEventThread =>
+          /* This event is suppose to be handled by the ControllerEventThread. In the future we change the ControllerEventThread to handle:
```

Comment:
Maybe we can file a JIRA and remove the comment?

Reply:
https://issues.apache.org/jira/browse/KAFKA-8409